x-pack/plugins/security_solution/server/lib/sourcerer/readme.md

```diff
@@ -0,0 +1,53 @@
+# Sourcerer API
+
+### Model reference
+
+```typescript
+interface KibanaDataView {
+  /** Uniquely identifies a Kibana Data View */
+  id: string;
+  /**  list of active patterns that return data  */
+  patternList: string[];
+  /**
+   * title of Kibana Data View
+   * title also serves as "all pattern list", including inactive
+   * comma separated string
+   */
+  title: string;
+}
+```
+
+### API usage
+
+The sourcerer API has one route with 2 methods
+
+1. POST - `createSourcererDataViewRoute`
+   1. This route is called from `security_solution/public/plugin.tsx` on app load. It passes an argument of `patternList` which is an array of the config index patterns defined in Stack Management > Advanced Settings > Security Solution > Elasticsearch indices along with the default signal index
```

Comment:
I think it worth to add that the numerated steps under the POST method describes the server side logic for `createSourcererDataViewRoute ` and add the description for the request/response as well.

Reply:
thats what steps 3-8 describe